coral-hive/src/main/java/com/linkedin/coral/transformers/ShiftArrayIndexTransformer.java

```diff
@@ -42,16 +42,18 @@ public boolean condition(SqlCall sqlCall) {
   @Override
   public SqlCall transform(SqlCall sqlCall) {
     final SqlNode itemNode = sqlCall.getOperandList().get(1);
+    SqlNode newIndex;
```

Comment:
could you add a todo for the unit test here so that the pending item is tracked?